tests/main.test.ts

```diff
@@ -279,6 +272,21 @@ describe("User start/stop", () => {
       }
     }
   });
+
+  test("User can't start an issue if they have previously been unassigned by an admin", async () => {
+    const issue = db.issue.findFirst({ where: { id: { equals: 2 } } }) as unknown as Issue;
+    const sender = db.users.findFirst({ where: { id: { equals: 2 } } }) as unknown as Sender;
+
+    const context = createContext(issue, sender, "/start");
+
+    try {
+      await userStartStop(context as unknown as Context);
+    } catch (error) {
+      if (error instanceof Error) {
+        expect(error.message).toEqual("This user was unassigned from this task previously. Cannot auto assign");
```

Comment:
I think it's better to use
```js
await expect(userStartStop(context as unknown as Context)).rejects.toThrow('This user was unassigned from this task previously. Cannot auto assign');
```
because currently if the error is not thrown or error is not Error, the test will just end and you won't now something's wrong. 
also fix this in other places

Reply:
I recently realized this with others working on the tests it's not ideal and I will avoid it future for sure